src/main/java/duke/task/Events.java

```diff
@@ -0,0 +1,62 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import duke.DukeException;
+
+/**
+ * Represents an event task.
+ */
+public class Events extends Task {
```

Comment:
Should this class be named "Event"? It represents a single event and not a collection of them.